www/apps/book/app/advanced-development/modules/module-links/page.mdx

```diff
@@ -0,0 +1,96 @@
+export const metadata = {
+  title: `${pageNumber} Module Link`,
+}
+
+# {metadata.title}
+
+In this chapter, you’ll learn what a module link is.
+
+<Note type="soon" title="In Development">
+
+Module links are in active development.
+
+</Note>
+
+## What is a Module Link?
+
+A module link forms an association between two data models of different modules, while maintaining module isolation.
+
+You can then retrieve data across the linked modules, and manage their linked records.
+
+---
+
+## How to Create a Module Link?
+
+Links are defined in a TypeScript or JavaScript file under the `src/links` directory. The file defines the link using the `defineLink` function imported from `@medusajs/utils` and exports it.
+
+For example:
+
+export const highlights = [
+  ["6", "links", "Special `links` property that holds the `HelloModule`'s data models."],
+  ["7", "links", "Special `links` property that holds the `ProductModule`'s data models."],
+]
+
+```ts title="src/links/hello-product.ts" highlights={highlights}
+import HelloModule from "../modules/hello"
+import ProductModule from "@medusajs/product"
+import { defineLink } from "@medusajs/utils"
+
+export default defineLink(
+  HelloModule.links.myCustom,
+  HelloModule.links.product
+)
+```
+
+The `defineLink` function accepts two parameters, which are the the data models to form the link between them.
```

Comment:
I think we'll need to figure out how we talk about the `links.[model]`, because it is not exactly pointing to the actual data models. Not important now, but something to think about.

Reply:
Agreed, I made a change [here](https://github.com/medusajs/medusa/pull/7918/commits/aa349a7486d04a772d949be93dfa726cd2dd55d0) to slightly explain it better. I think this explanation will also align if we rename `links` to `linksConfig` later.